### ISSUE TITLE
Failure when adding dependencies: ghc: needed (==8.0.1)

### ISSUE THREAD

**Issue by andreasthoelke** [begriffs/haskell-vim-now#225]
When installing on macOS 10.12.2 I receive the following error:

```
-- Failure when adding dependencies:
      ghc: needed (==8.0.1), 8.0.1.20161117 found
    needed for package apply-refact-0.3.0.0
*** Binary installation failed with error 1.
```

Do I actually have to downgrade ghc?

**Comment by andreasthoelke** [comment]
This is solved by just upgrading ghc to 8.0.2. It's just that the message `needed (==8.0.1)` is a bit confusing.

**Comment by begriffs** [comment]
Weird that it just started having problems because it comes from #152 which was merged early in 2016.

Maybe it's because your global Stack config specifies an LTS version that conflicted with `apply-refact`. The vim installer searches in your config file and matches the system LTS version so that the installation of `ghc-mod` will function properly (it is sensitive to cabal and ghc version). https://github.com/begriffs/haskell-vim-now/blob/master/scripts/setup.sh#L44
